perses/app/relative_point_mutation_setup.py

```diff
@@ -123,59 +114,83 @@ def __init__(self,
+            water_model : str, default 'tip3p'
+                solvent model to use for solvation
+            ionic_strength : float * unit.molar, default 0.15 * unit.molar
+                the total concentration of ions (both positive and negative) to add using Modeller.
+                This does not include ions that are added to neutralize the system.
+                Note that only monovalent ions are currently supported.
             forcefield_files : list of str, default ['amber14/protein.ff14SB.xml', 'amber14/tip3p.xml']
                 forcefield files for proteins and solvent
             barostat : openmm.MonteCarloBarostat, default openmm.MonteCarloBarostat(1.0 * unit.atmosphere, 300 * unit.kelvin, 50)
                 barostat to use
-            forcefield_kwargs : dict, default {'removeCMMotion': False, 'ewaldErrorTolerance': 1e-4, 'nonbondedMethod': app.NoCutoff, 'constraints' : app.HBonds, 'hydrogenMass' : 4 * unit.amus}
+            forcefield_kwargs : dict, default {'removeCMMotion': False, 'ewaldErrorTolerance': 1e-4, 'constraints' : app.HBonds, 'hydrogenMass' : 4 * unit.amus}
                 forcefield kwargs for system parametrization
-            periodic_forcefield_kwargs : dict default {'nonbondedMethod':app.PME}
+            periodic_forcefield_kwargs : dict, default {'nonbondedMethod': app.PME}
+                periodic forcefield kwargs for system parametrization
+            nonperiodic_forcefield_kwargs : dict, default None
+                non-periodic forcefield kwargs for system parametrization
             small_molecule_forcefields : str, default 'gaff-2.11'
                 the forcefield string for small molecule parametrization
 
-        TODO : allow argument for separate apo structure if it exists separately
-               allow argument for specator ligands besides the 'ligand_filename'
+        TODO : allow argument for spectator ligands besides the 'ligand_filename'
+
         """
-        from openforcefield.topology import Molecule
-        from openmmforcefields.generators import SystemGenerator
 
-        # first thing to do is make a complex and apo...
-        pdbfile = open(receptor_filename, 'r')
-        pdb = app.PDBFile(pdbfile)
-        pdbfile.close()
-        receptor_positions, receptor_topology, receptor_md_topology = pdb.positions, pdb.topology, md.Topology.from_openmm(pdb.topology)
-        receptor_topology = receptor_md_topology.to_openmm()
-        receptor_n_atoms = receptor_md_topology.n_atoms
+        # First thing to do is load the apo protein to mutate...
+        protein_pdbfile = open(protein_filename, 'r')
+        protein_pdb = app.PDBFile(protein_pdbfile)
+        protein_pdbfile.close()
+        protein_positions, protein_topology, protein_md_topology = protein_pdb.positions, protein_pdb.topology, md.Topology.from_openmm(protein_pdb.topology)
+        protein_topology = protein_md_topology.to_openmm()
+        protein_n_atoms = protein_md_topology.n_atoms
```

Comment:
Are we assuming that the protein is always passed in already protonated?

Reply:
yes. i could a `_protonate` function if we want?

Reply:
I think it's fine as long as it's documented that we expect it protonated, I was just worried that `protein_n_atoms` might change

Reply:
there is a single template for _all_ protein amino acids. if the starting residue cannot be matched, then it will throw error...i was holding off on generalizing this until biopolymer residues were incorproated into openforcefield..., but i dont know when that will be